extras/rya.pcj.fluo/pcj.fluo.api/src/main/java/org/apache/rya/indexing/pcj/fluo/api/CreatePcj.java

```diff
@@ -82,6 +82,7 @@
  */
 @DefaultAnnotation(NonNull.class)
 public class CreatePcj {
+    private static final Logger log = Logger.getLogger(CreatePcj.class);
```

Comment:
Which logging framework are we using? I can update my code at the very least to use that. Is it SLF4J?